packages/babel-polyfill-udemy-website/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "babel-polyfill-udemy-website",
-  "version": "7.0.4",
```

Comment:
It wasn't really a bug fix release. I was more of a new feature release. Oh well.